Lib/test/test_bytes.py

```diff
@@ -459,6 +459,12 @@ def test_fromhex(self):
         self.assertRaises(ValueError, self.type2test.fromhex, '\x00')
         self.assertRaises(ValueError, self.type2test.fromhex, '12   \x00   34')
 
+        # For odd number of character(s)
+        for value in ("a", "aaa", "deadbee"):
```

Comment:
nit: would be nice to have a test case where the string is of even length, e.g.  " aaa" and test cases where we check the error message for " aa a " / " aa a a "

Reply:
sorry for going ahead and merging @hauntsaninja. I should have waited for more input from you.

Reply:
https://github.com/python/cpython/pull/127818

Reply:
oh all good, was just a nit and commits are cheap :-) thanks for reviewing!